Traceback (most recent call last):
  File "C:\Users\rob_t\anaconda3\envs\steam_nlp\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\rob_t\anaconda3\envs\steam_nlp\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\rob_t\anaconda3\envs\steam_nlp\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\rob_t\anaconda3\envs\steam_nlp\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\rob_t\anaconda3\envs\steam_nlp\lib\asyncio\base_events.py", line 646, in run_until_complete
    return future.result()
  File "C:\Users\rob_t\anaconda3\envs\steam_nlp\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\rob_t\anaconda3\envs\steam_nlp\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\rob_t\anaconda3\envs\steam_nlp\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
fig, ax = plt.subplots(figsize=(9,7))
ax=sns.countplot(
    data=pd.melt(df.loc[:,'Sentiment':]),
    hue="value",
    x="variable",
    hue_order=['Negative','Positive','Neutral'],
    palette=['#FF6F69','#88D8B0','#ffcc5c'],)
ax.set_xticklabels(['Before', 'After'], size=12)
ax.set_xlabel('Lexicon Modification', size=12)
ax.set_title("Compared Sentiment Distribution", size =14)
plt.legend(title='Sentiment', title_fontsize=12, loc='upper center')
plt.ylim(0,30000)

# create labels
for c in ax.containers:
    labels = [f'\n\n {h/df.Sentiment.count()*100:0.1f}%' if (h := v.get_height()) > 0 else '' for v in c]
    ax.bar_label(c, labels=labels, label_type='center')
ax.bar_label(ax.containers[0], label_type='center')
ax.bar_label(ax.containers[1], label_type='center')
ax.bar_label(ax.containers[2], label_type='center');
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
Input [1;32mIn [4][0m, in [0;36m<cell line: 1>[1;34m()[0m
[1;32m----> 1[0m fig, ax [38;5;241m=[39m [43mplt[49m[38;5;241m.[39msubplots(figsize[38;5;241m=[39m([38;5;241m9[39m,[38;5;241m7[39m))
[0;32m      2[0m ax[38;5;241m=[39msns[38;5;241m.[39mcountplot(
[0;32m      3[0m     data[38;5;241m=[39mpd[38;5;241m.[39mmelt(df[38;5;241m.[39mloc[:,[38;5;124m'[39m[38;5;124mSentiment[39m[38;5;124m'[39m:]),
[0;32m      4[0m     hue[38;5;241m=[39m[38;5;124m"[39m[38;5;124mvalue[39m[38;5;124m"[39m,
[0;32m      5[0m     x[38;5;241m=[39m[38;5;124m"[39m[38;5;124mvariable[39m[38;5;124m"[39m,
[0;32m      6[0m     hue_order[38;5;241m=[39m[[38;5;124m'[39m[38;5;124mNegative[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124mPositive[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124mNeutral[39m[38;5;124m'[39m],
[0;32m      7[0m     palette[38;5;241m=[39m[[38;5;124m'[39m[38;5;124m#FF6F69[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124m#88D8B0[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124m#ffcc5c[39m[38;5;124m'[39m],)
[0;32m      8[0m ax[38;5;241m.[39mset_xticklabels([[38;5;124m'[39m[38;5;124mBefore[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124mAfter[39m[38;5;124m'[39m], size[38;5;241m=[39m[38;5;241m12[39m)

[1;31mNameError[0m: name 'plt' is not defined
NameError: name 'plt' is not defined

